exporter/honeycombexporter/logs_exporter.go

```diff
@@ -0,0 +1,26 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package honeycombexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/honeycombexporter"
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.uber.org/zap"
+)
+
+type honeycombLogsExporter struct {
+	logger *zap.Logger
```

Comment:
We should include a field to hold the markers.  In `newLogsExporter` we should use the markers from the `cfg` to set the field on the exporter's struct.